### PR TITLE
Update CSS handles

### PR DESCRIPTION
**Pull request by iaronaraujo**
#### What is the purpose of this pull request?

Add new CSS handles and update some of the old ones

#### How should this be manually tested?

[storecomponents](https://iaronaraujo--storecomponents.myvtex.com/top?_q=top&map=ft&order=OrderByPriceASC)
[alssports](https://iaronaraujo--alssports.myvtex.com/clothing?map=c,specificationFilter_7721&query=/clothing/Boys)
[tbb](https://iaronaraujo--tbb.myvtex.com/cuidados-para-a-pele/hidratante)
[samsungar](https://iaronaraujo--samsungar.myvtex.com/galaxy?_q=galaxy&map=ft&page=3)

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/NotFoundLayout.js [hunk 1]
```diff
@@ -25,13 +24,14 @@ const NotFoundLayout = ({
...
+  const handles = useCssHandles(CSS_HANDLES)
 
   return (
     <SearchPageContext.Provider value={context}>
       <div
-        className={`${generateBlockClass(
-          styles['notFound--layout'],
-          blockClass
+        className={`${applyModifiers(
+          handles.notFound,
```
Comment (jgfidelis) [inline react/NotFoundLayout.js]: isso aqui vai dar problema pois eu criei as classes `notFound--layout` justamente para não colocar `notFound` pq a classe notFound tem estilos de grid e colocar ela faz esses estilos serem aplicados, o que não é desejado no SR flexível. O applyModifiers se não me engano vai colocar a classe `notFound notFound--layout` o que não é desejado. Isso serve para todas as classes com --layout nesse repositório.

File: docs/README.md [hunk 2]
```diff
@@ -470,69 +470,79 @@ To use this CSS API, you must add the `styles` builder and create an app styling
...
-| `orderByDropdown`                  | the dropdown that appears when the "Sort By" button found on search results is pressed | [SelectionListOrderBy](/react/components/SelectionListOrderBy.js)         |
-| `orderByOptionsContainer`          | the container with the "Order by" options of the "Sort by" button                      | [SelectionListOrderBy](/react/components/SelectionListItem.js)            |
-| `orderByOptionItem`                | the "Order by" option that appears in the container of the "Sort by" button            | [SelectionListOrderBy](/react/components/SelectionListItem.js)            |
-
-| `categoriesContainer` | The container for the department filters | [DepartmentFilters](/react/components/DepartmentFilters.js) |
-| `categoryGroup` | Container for each category group in the department filters | [CategoryFilter](/react/components/CategoryFilter.js) |
-| `categoryParent` | View of the parent category of this group | [CategoryFilter](/react/components/CategoryFilter.js) |
+| Token name                            |
```
Comment (klzns) [inline docs/README.md]: Use CSS Handles
```suggestion
| CSS Handles                            | 
```

File: docs/README.md [hunk 3]
```diff
@@ -470,69 +470,79 @@ To use this CSS API, you must add the `styles` builder and create an app styling
 
 Below, we describe the namespaces that are defined in the search-result.
```
Comment (klzns) [inline docs/README.md]:
```suggestion
```

File: docs/README.md [hunk 4]
```diff
@@ -470,69 +470,79 @@ To use this CSS API, you must add the `styles` builder and create an app styling
```
Comment (klzns) [inline docs/README.md]: Fix title. Use "Customization"

File: react/NotFoundSearch.js [hunk 5]
```diff
@@ -2,35 +2,54 @@ import PropTypes from 'prop-types'
...
     <Fragment>
       <div
-        className={`${
-          searchResult.searchNotFound
-        } flex flex-column-s flex-row-ns justify-center-ns items-center h-auto-s h5-ns`}
+        className={`${handles.searchNotFound} flex flex-column-s flex-row-ns justify-center-ns items-center h-auto-s h5-ns`}
       >
-        <div className="flex justify-end-ns justify-center-s ttu f1 ph4 pv4-s pv0-ns c-muted-3 ph9 b">
+        <div
+          className={` ${handles.searchNotFoundOops} flex justify-end-ns justify-center-s ttu f1 ph4 pv4-s pv0-ns c-muted-3 ph9 b`}
```
Comment (klzns) [inline react/NotFoundSearch.js]:
```suggestion
          className={`${handles.searchNotFoundOops} flex justify-end-ns justify-center-s ttu f1 ph4 pv4-s pv0-ns c-muted-3 ph9 b`}
```

File: react/components/loaders/LoadingSpinner.js [hunk 6]
```diff
@@ -1,12 +1,21 @@
...
   const { loading } = props
+  const handles = useCssHandles(CSS_HANDLES)
   return (
     loading && (
-      <div className="w-100 flex justify-center">
-        <div className="w3 ma0">
+      <div
+        className={`${handles.loadingSpinnerOuterContainer} w-100 flex justify-center`}
+      >
+        <div className={`${handles.loadingSpinnerOuterContainer} w3 ma0`}>
```
Comment (klzns) [inline react/components/loaders/LoadingSpinner.js]: Ops should be inner here
```suggestion
        <div className={`${handles.loadingSpinnerInnerContainer} w3 ma0`}>
```

File: react/components/AccordionFilterContainer.js [hunk 7]
```diff
@@ -3,15 +3,19 @@ import PropTypes from 'prop-types'
...
 import AccordionFilterGroup from './AccordionFilterGroup'
-
-import styles from '../searchResult.css'
 import AccordionFilterPriceRange from './AccordionFilterPriceRange'
 
 const CATEGORIES_TITLE = 'store/search.filter.title.categories'
+const CSS_HANDLES = [
+  'accordionFilterItemOptions',
+  'accordionFilter',
+  'filterAccordionBreadcrumbs',
```
Comment (klzns) [inline react/components/AccordionFilterContainer.js]: Those 3 classes have CSS definitions in `searchResult.css`

File: react/components/FacetCheckboxList.js [hunk 8]
```diff
@@ -1,20 +1,23 @@
 import classNames from 'classnames'
 import React from 'react'
 import { Checkbox } from 'vtex.styleguide'
+import { useCssHandles } from 'vtex.css-handles'
 
 import useSelectedFilters from '../hooks/useSelectedFilters'
-import styles from '../searchResult.css'
+
+const CSS_HANDLES = ['filterAccordionItemBox']
```
Comment (klzns) [inline react/components/FacetCheckboxList.js]: Have a CSS definition

File: react/components/FilterSidebar.js [hunk 9]
```diff
@@ -18,7 +19,12 @@ import useFacetNavigation, {
   buildQueryAndMap,
 } from '../hooks/useFacetNavigation'
 
-import searchResult from '../searchResult.css'
+const CSS_HANDLES = [
+  'filterPopupButton',
```
Comment (klzns) [inline react/components/FilterSidebar.js]: Have CSS definition

File: react/components/FilterSidebar.js [hunk 10]
```diff
@@ -18,7 +19,12 @@ import useFacetNavigation, {
...
+  'filterPopupTitle',
+  'filterPopupArrowIcon',
+  'filterButtonsBox',
```
Comment (klzns) [inline react/components/FilterSidebar.js]: Same

File: react/components/SelectionListOrderBy.js [hunk 11]
```diff
@@ -5,13 +5,22 @@ import classNames from 'classnames'
...
 import { useDevice } from 'vtex.device-detector'
+import { useCssHandles } from 'vtex.css-handles'
 
 import SelectionListItem from './SelectionListItem'
 import useOutsideClick from '../hooks/useOutsideClick'
-import searchResult from '../searchResult.css'
+
+const CSS_HANDLES = [
+  'orderByButton',
+  'orderByOptionsContainer',
```
Comment (klzns) [inline react/components/SelectionListOrderBy.js]: CSS definitions

File: react/FilterNavigator.js [hunk 12]
```diff
@@ -97,9 +101,10 @@ const FilterNavigator = ({
     <Fragment>
       <div className={filterClasses}>
         <div
-          className={`${styles['filter__container']} ${
-            styles['filter__container--title']
-          } bb b--muted-4`}
+          className={`${applyModifiers(
```
Comment (klzns) [inline react/FilterNavigator.js]: Missing `filter__container`, I could check looking at the snapshot

**Pull request merged**